### PR TITLE
Added tickets closing news

### PR DESCRIPTION
**Pull request by swapnilogale**
Added news item for:
a. Tickets closing
b. Welcome wagon

### CODE REVIEW

**Comment by swapnilogale** [comment]
@ericholscher @plaindocs this looks ok to me - just pushing a news about tickets closing, so that it triggers an email from Mailchimp

File: docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+G'day everyone!
+
+We're just over a week away from the start of the conference,
+and we're getting excited to invite you into our online venue for 2 days of content, collaboration, and conversation.
+We will be closing ticket sales this **Friday, 27th November**, so if you've been on the fence, now is the time to buy one!
+
+If you were hoping to come to this years event,
+we recommend that you `get your tickets soon <https://www.writethedocs.org/conf/australia/2020/tickets/>`_.
+
+We have done our internal testing of the platform, and we're getting everything ready for prime time.
```
Comment (plaindocs) [inline docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst:18]: This all reads a little awkward. Not enough to block, but if you fancy rewriting...

**Review by plaindocs — APPROVED** [review]

File: docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Thanks so much to all our sponsors
+----------------------------------
+
+We are so grateful to have our sponsors help in bringing these events to life every year.
+
+Thanks so much to the following companies for supporting the Australia and India conference this year:
+
+.. datatemplate::
+   :source: /_data/{{shortcode}}-{{year}}-config.yaml
+   :template: {{year}}/sponsors-simplelist.rst
```
Comment (ericholscher) [inline docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst:51]: We have Red Hat to add here, I'll do that in another PR, but we should do it prior to shipping.

**Review by ericholscher — APPROVED** [review]
👍 

Looks good, let me do some quick bookeeping on sponsors, and then we'll be good to go.

File: docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst (hunk 2 above)
Comment (ericholscher) [reply docs/conf/australia/2020/news/tickets-closing-welcome-wagon.rst:51]: RH PR is merged 👍

**Pull request merged**